### PR TITLE
Remove " ' " from WordPress' word.

### PR DESCRIPTION
**Pull request by jankimoradiya**
Removed extra single quote from " WordPress' " word.

### CODE REVIEW

File: philosophy.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Whenever possible, commands should emulate one another with consistent usage beh
 
 ## Decisions, not options.
 
-WordPress' philosophy [states](https://wordpress.org/about/philosophy/):
+WordPress philosophy [states](https://wordpress.org/about/philosophy/):
```
Comment (schlessera) [inline philosophy.md]: This apostrophe is actually correct, as it is a possessive phrase.

File: shell-friends.md [hunk 2]
```diff
@@ -176,7 +176,7 @@ Explanation
 
 Add this function to your ~/.bashrc are reload your shell (or open a new shell).
 You'll need to substitute these database credentials with your own.
-When you need create a new wordpress install, call this function and specify the
+When You'll need to create a new site and install wordpress, call this function and specify the
```
Comment (schlessera) [inline shell-friends.md]: Your suggestion has other issues as well, but the original sentence is indeed missing a `to` and has wrong casing for WordPress.

Here's what I think is a good fix:
```suggestion
When you need to create a new WordPress site, call this function and specify the
```

**Comment by jankimoradiya** [comment]
Hello @schlessera,

Thank you for the feedback. 
I have updated sentence as per your suggestion & create PR for it.

Thanks

File: philosophy.md (hunk 1 above)
Comment (schlessera) [reply philosophy.md]: You still need to remove this change, the apostrophe should stay.

File: shell-friends.md [hunk 3]
```diff
@@ -176,7 +176,7 @@ Explanation
 
 Add this function to your ~/.bashrc are reload your shell (or open a new shell).
 You'll need to substitute these database credentials with your own.
-When you need create a new wordpress install, call this function and specify the
+When You need to create a new WordPress site, call this function and specify the
```
Comment (schlessera) [inline shell-friends.md]: Lowercase `you` here.
```suggestion
When you need to create a new WordPress site, call this function and specify the
```

**Comment by jankimoradiya** [comment]
Hello @schlessera,

I have updated changes as per your suggestion & create PR for it.

Thanks

**Comment by schlessera** [comment]
Thanks, this is fine for merging now.

**Comment by schlessera** [comment]
Thanks, this is fine for merging now.

**Pull request merged**